config/filter.d/apache-auth.conf

```diff
@@ -12,10 +12,10 @@ before = apache-common.conf
 
 failregex = ^%(_apache_error_client)s (AH(01797|01630): )?client denied by server configuration: (uri )?\S*(, referer: \S+)?\s*$
             ^%(_apache_error_client)s (AH01617: )?user .*? authentication failure for "\S*": Password Mismatch(, referer: \S+)?$
-            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?\S*(, referer: \S+)?\s*$
-            ^%(_apache_error_client)s (AH01614: )?client used wrong authentication scheme: \S*(, referer: \S+)?\s*$
+            ^%(_apache_error_client)s (AH01618: )?user .*? not found(: )?.*(, referer: \S+)?\s*$
```

Comment:
> A greedy match (...) could be manipulated with referer, so allows injecting on referer or other data before...

As we have `referer: \S+` in the rule, I'm not sure injection is possible, as referer won't be able to use spaces. Don't you think so ?